pkg/data/client/error.go

```diff
@@ -4,11 +4,11 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
-var InvalidConfigError = &microerror.Error{
-	Kind: "InvalidConfigError",
+var invalidConfigError = &microerror.Error{
```

Comment:
Discovered a mistake in my previous PR. This is supposed to be private, as mentioned in [fmt](https://github.com/giantswarm/fmt/blob/master/go/errors.md#custom-errors)